sdcm/logcollector.py

```diff
@@ -469,14 +469,29 @@ class GrafanaSnapshot(GrafanaEntity):
     Extends:
         GrafanaEntity
     """
-
+    panels_load_timeout = 10
+    scroll_step = 1000
     snapshot_locators_sequence = [
         (By.XPATH, """//button[contains(@class, "navbar-button--share")]"""),
         (By.XPATH, """//ul/li[contains(text(), "Snapshot")]"""),
         (By.XPATH, """//button//span[contains(text(), "Publish to snapshot.raintank.io")]"""),
         (By.XPATH, """//a[contains(@href, "https://snapshot.raintank.io")]""")
     ]
 
+    def scrolldown_dashboards_view(self, remote_browser):
+        scroll_ready_element = (By.XPATH, """//span[contains(text(), "Total Nodes")]""")
+        scrolling_element = (By.XPATH, "//div[@class='view']")
+
+        WebDriverWait(remote_browser, 60).until(EC.visibility_of_element_located(scroll_ready_element))
+
+        element = remote_browser.find_element(*scrolling_element)
+
+        scrolling_height = remote_browser.execute_script("return arguments[0].scrollHeight", element)
+
+        for scroll_size in range(0, scrolling_height, self.scroll_step):
+            remote_browser.execute_script(f'arguments[0].scrollTop = {scroll_size}', element)
+            time.sleep(self.panels_load_timeout)
+
```

Comment:
Consider to make the naming more consistent:

- `scroll_step`
- `scroll_ready_element`
- `scroll_size`

but

- `scrolling_element`
- `scrolling_height`

Reply:
done